install/windows/build-image.py

```diff
@@ -18,28 +18,26 @@ def main():
     )
 
     print("Saving container image")
-    subprocess.run(
+    cmd = subprocess.Popen(
         [
             "docker",
             "save",
             "dangerzone.rocks/dangerzone",
-            "-o",
-            "share/container.tar",
-        ]
+        ],
+        stdout=subprocess.PIPE,
```

Comment:
Using `subprocess.run()`, we would wait for the command to complete. There are two problems with that:

1. If the command writes to a pipe that no one reads from, it will fill after 65KiB of data, and the command will block.
2. Even if the pipe didn't fill, we have more performance gains if we run `docker save` on parallel with `gzip`.

Reply:
Interesting! Good to know :)